### ISSUE TITLE
Change content rating of the app

### ISSUE THREAD

**Issue by sivaraam** [commons-app/apps-android-commons#5656]
It was brought up via a user feedback that the app should be rated as 18+ as it bring up adult media [[ref](https://commons.wikimedia.org/wiki/Commons_talk:Mobile_app#App_Content_must_be_ratted_18+)]. This is indeed true as the app doesn't do anything to restrict the such images in Explore / search. So, it makes sense to bump the content rating of our play store listing.

OTOH, it would be great if we have some way to filter such content so that users have the flexibility to choose what kind of content they're shown. That would also help us to bump down out content rating so that the app is accessible to wide age range of users.

**Comment by sivaraam** [comment]
BTW, while I checked the Play console, we do seem to mention the target audience as 18+

![image](https://github.com/commons-app/apps-android-commons/assets/12448084/37ec930b-9647-421f-b2c0-cf368a90b53a)

The [play store listing](https://play.google.com/store/apps/details?id=fr.free.nrw.commons&pli=1) seem to mention the app as 12+, though. 

![image](https://github.com/commons-app/apps-android-commons/assets/12448084/507fa209-a8ad-4653-8204-499b1c9cc295)

Any idea on this @nicolas-raoul ?

**Comment by nicolas-raoul** [comment]
Maybe homogeneize to 18?

**Comment by pigsonthewing** [comment]
>  some way to filter such content so that users have the flexibility to choose what kind of content they're show

What criteria will be used?

**Comment by nicolas-raoul** [comment]
I agree with @pigsonthewing, this is a huge can of worm if we start policing what is safe or not. And even if we can establish such a policy, there will always occasional uncategorized unsafe uploads which will prevent us from bumping down out content rating. We are a post-moderated platform for anonymous people to upload pictures, this is intrinsically unsafe.

But the Wikimedia Commons website ever implements such a system, I am happy to follow.

**Comment by sivaraam** [comment]
> Maybe homogeneize to 18?

Makes sense. I filled up the Content rating questionarre and this is where we are now:

![image](https://github.com/commons-app/apps-android-commons/assets/12448084/29e0a8b3-9bfb-4438-a5b4-d3c9d0ff0f35)

<details><summary>Old rating for reference</summary>
<p>

![image](https://github.com/commons-app/apps-android-commons/assets/12448084/aa314bbc-5a19-416f-afd3-5ed9e263ecd9)


</p>
</details> 

We got 18+ in most cases but we seem to have bumped down to "All ages" for Brazil (used to be 14+) and 12+ for Germany (used to be 18+). Not sure the reason behind this. I've submitted the change for review. Let's see how it goes.

**Comment by sivaraam** [comment]
> I agree with @pigsonthewing, this is a huge can of worm if we start policing what is safe or not. And even if we can establish such a policy, there will always occasional uncategorized unsafe uploads which will prevent us from bumping down out content rating. We are a post-moderated platform for anonymous people to upload pictures, this is intrinsically unsafe.
> 
> But if the Wikimedia Commons website ever implements such a system, I am happy to follow.

Most certainly. I too didn't have an idea wherein the app would enable such filtering to enable lower content rating. That cannot scale. A system that's adopted by Wikimedia Commons is what would be ideal for us too 🙂 

There seem to have been discussion about this [ [ref 1](https://commons.wikimedia.org/wiki/Commons:Sexual_content/April_2010)  ] [ [ref 2](https://commons.wikimedia.org/wiki/Commons:Sexual_content) ]. There doesn't seem any commonly adopted policy, though.

**Comment by sivaraam** [comment]
> We got 18+ in most cases but we seem to have bumped down to "All ages" for Brazil (used to be 14+) and 12+ for Germany (used to be 18+). Not sure the reason behind this. I've submitted the change for review. Let's see how it goes.

The rating seems live now in Play store:

![Screenshot_2024-03-31-09-33-43-44_d0ef7a9810af20f7b394f4f927b723b0_1](https://github.com/commons-app/apps-android-commons/assets/12448084/578c46fc-5fb3-4f01-9366-231fe5bec73e)

Let me know if we could correct the rating for Germany and Brazil (or) kindly mention if they are appropriate ratings for those countries.

**Comment by deryckchan** [comment]
I'm not sure 18+ is the correct rating to go for, even if out of precaution. The Wikipedia app is rated 3+ / all ages, and the same supposedly sensitive content from Commons would be available on the Wikipedia app too.

**Comment by nicolas-raoul** [comment]
Thanks Deryck for pointing this out!
I suggest we leave 18+ until we are confident enough to lower it to what we believe is the most appropriate value.
I guess the Wikipedia app developers have documented their rationale somewhere? Finding this document would help us a lot.

**Comment by deryckchan** [comment]
I've sent an email to the [official Wikipedia app helpline](https://www.mediawiki.org/wiki/Wikimedia_Apps/Android_FAQ) to see if I can get in touch with the maintainers on their side.

**Comment by sivaraam** [comment]
> The Wikipedia app is rated 3+ / all ages, and the same supposedly sensitive content from Commons would be available on the Wikipedia app too.

That is indeed a curious observation. It would certainly be helpful to know how they have such a content rating.

> I've sent an email to the [official Wikipedia app helpline](https://www.mediawiki.org/wiki/Wikimedia_Apps/Android_FAQ) to see if I can get in touch with the maintainers on their side.

Thank you for this! Let's hope we get a helpful response from them. Feel free to let us know if yoi don't seem to receive a proper response from them. We'll explore other ways of reaching out to them.

**Comment by RitikaPahwa4444** [comment]
This time our app got rejected because of this reason:

> We found an issue in the following area(s):
> Application Ratings: Please see attached screenshot

![Image](https://github.com/user-attachments/assets/3edb65ce-a3f7-4c43-ac7f-44eb9e09ad89)

Wondering why the app got rejected; we had this rating already 🤔

**Comment by nicolas-raoul** [comment]
That's not even a screenshot haha... worth an appeal I would say.

**Comment by sivaraam** [comment]
I think it's worth exploring a bit before appealing. AFAIK, Google doesn't encourage appeals much. Given that this is the second rejection, we should better do our part of research before appealing.

From my cursory look, ot does seem we might somewhere conveying our app is usable by people less than 18 regardless our rating. [ [ref](https://stackoverflow.com/q/56667245/5614968) ]. We should check to see what we're doing wrong.

**Comment by deryckchan** [comment]
I got a templated response from the Wikipedia app helpline saying they're
looking into it, and then silence. :(

**Comment by sivaraam** [comment]
> I got a templated response from the Wikipedia app helpline saying they're
> looking into it, and then silence. :(

The WMF team once initiated a discussion with us back in May 2024 regarding the app's content rating (this is the WMF thread I refer in [this comment](https://github.com/commons-app/apps-android-commons/issues/6178#issuecomment-2661999130) ). This is likely owing to your e-mail to them :-)

**Comment by RitikaPahwa4444** [comment]
I tried modifying the answers to the questionnaire a bit but the rating doesn't seem to change. Checked the target audience and the tags used as well. Has anyone figured out what we might be doing wrong here? @sivaraam do we have a way to ask clarifying questions if appeals aren't encouraged?

**Comment by deryckchan** [comment]
Has the Wikipedia app maintainers given y'all any info about what they put
in the questionnaire?

**Comment by RitikaPahwa4444** [comment]
@deryckchan they feel based on our answers to the questionnaire, Google's judgement is likely to be correct. Regarding why Wikipedia and Commons app could have different ratings:

> it seems that there are some key differences between the Commons and Wikipedia app's, and the nature of the content on those projects, the degree of editorialisation of the content, as well the sites purpose and policies, that means there will likely always be some difference in the age rating between the applications. 

They suggested to bump up the target audience to 18+, which we have done already.

**Comment by mnalis** [comment]
> OTOH, it would be great if we have some way to filter such content so that users have the flexibility to choose what kind of content they're shown. That would also help us to bump down out content rating so that the app is accessible to wide age range of users.

related:
- https://github.com/commons-app/apps-android-commons/issues/5138

**Comment by sivaraam** [comment]
Thanks for the nudge @mnalis!

Owing to recent changes done to adjust the app to successfully pass the Play store policy review (see #6228), the current content rating of the app is 3+. Specifically, the explore and peer review have passed paly store review and are currently in the production track. So, I think we're good to go here. :-)

**Comment by deryckchan** [comment]
Well done folks!

**Comment by mnalis** [comment]
> Specifically, the explore and peer review have passed paly store review and are currently in the production track. So, I think we're good to go here. :-)

That's great news @sivaraam, so hopefully this issue can then be closed soon!

**Comment by sivaraam** [comment]
Yeah. I'll close it now. Feel free to reopen if you feel anything else is left to handle here. :-)
